docs/evaluate/temporal-cloud/pricing.mdx

```diff
@@ -77,7 +77,7 @@ They track billable operations within the Temporal Cloud Service, such as starti
 - **Side Effect recorded**.
   For a mutable [Side Effect](/workflows#side-effect), an Action occurs only when the value changes.
   (Be aware that some SDKs don't support Side Effects.)
-- [**NEW**] **Workflow Update**. [Workflow Updates](/encyclopedia/workflow-message-passing#sending-updates) is a primitive that combines a Signal and Query together for a single Action.
+- **Workflow Update received**. An [Update](/encyclopedia/workflow-message-passing#sending-updates) is received when you handle it or when your [Update Validator](/encyclopedia/workflow-message-passing#update-validators) rejects it.
```

Comment:
`processed`? I use processed for "accepted/rejected" on server side.

Reply:
@drewhoskins-temporal Is it fair to say that "An Action occurs when an [Update](....) is received, when you handle it, or when your [Update Validator](....) rejects an update"?

Reply:
(And if so, does that mean that you get 2 actions -- receive and handle, or 2 actions -- receive and reject?)

Reply:
@alexshtin 
Signal says "sent" 
Queries says "received"
I don't want Updates to use a _third_ word like "processed." 

I would prefer to use the same word for all of them if it's technically accurate.